### DISCUSSION TITLE
couldn't find config_compilers.xml

### DISCUSSION THREAD

**Discussion by jingyizhuo** [E3SM-Project/E3SM#5855] [E3SM model help]
I am installing CESM2 on Perlmutter, and I'm using the following GitHub repository as a reference for building the machine: https://github.com/E3SM-Project/E3SM/tree/master/cime_config/machines.

While I have successfully obtained the [config_batch.xml](https://github.com/E3SM-Project/E3SM/blob/master/cime_config/machines/config_batch.xml) and [config_machines.xml](https://github.com/E3SM-Project/E3SM/blob/master/cime_config/machines/config_machines.xml) files, I am now seeking assistance in finding the relevant config_compilers.xml file suitable for Perlmutter.

I would be sincerely grateful for any help you can provide!

**Comment by rljacob** [comment]
We aren't using config_compilers.xml any more on master.  You can find a Perlmutter example on the maint-1.0 branch.  https://github.com/E3SM-Project/E3SM/blob/maint-1.0/cime/config/e3sm/machines/config_compilers.xml

**Comment by jingyizhuo** [comment]
Thank you for your prompt response! I can run CESM2 on Perlmutter now, and only using the default settings of the gnu compiler works.
